### PR TITLE
SER-64 golang auth implement symfony guard

### PR DESCRIPTION
**Pull request by GemTay**

### CODE REVIEW

**Review by TomTucka — APPROVED** [review]
🚀

File: serve-web/src/Controller/SecurityController.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    public function __construct(UserProvider $userProvider)
+    {
+        $this->userProvider = $userProvider;
+    }
+    /**
+     * @Route("/login", name="app_login")
+     */
+    public function login(AuthenticationUtils $authenticationUtils): Response
+    {
+        // if ($this->getUser()) {
```
Comment (coderste) [inline serve-web/src/Controller/SecurityController.php:24]: Can we remove this comment?

File: serve-web/templates/security/login.html.twig [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+            </form>
+        </div>
+    </div>
+
+    <p>
+        <a href="{{ path('password-reset-request') }}" class="behat-link-password-reset">Forgotten your password?</a>
+    </p>
+
+    {#
```
Comment (coderste) [inline serve-web/templates/security/login.html.twig:77]: Again can we just remove this comment?

**Review by coderste — COMMENTED** [review]
2 comments on commented out code that could be removed maybe?

**Pull request merged**